content/post/kubedb-v2024.01.31/index.md

```diff
@@ -0,0 +1,544 @@
+
+Here's a sample manifest to provision FerretDB with User provided backend
+
+```yaml
+apiVersion: kubedb.com/v1alpha2
+kind: FerretDB
+metadata:
+  name: ferretdb-sample
+  namespace: demo
+spec:
+  replicas: 1
+  version: "1.18.0"
+  authSecret:
+    externallyManaged: true
+    name: ha-postgres-auth
+  storage:
+    storageClassName: "standard"
+    accessModes:
+    - ReadWriteOnce
+    resources:
+      requests:
+        storage: 600Mi  
+  backend:
+    externallyManaged: true
+    postgres:
+      url: postgres://ha-postgres.demo.svc.cluster.local:5432/
+  terminationPolicy: WipeOut
+```
+
+Here's another sample manifest using KubeDB provided and managed backend
+
+```yaml
+apiVersion: kubedb.com/v1alpha2
+kind: FerretDB
+metadata:
+  name: ferredb-sample-2
+  namespace: demo
+spec:
+  version: "1.18.0"
+  terminationPolicy: WipeOut
+  replicas: 1
+  sslMode: disabled  
+  storage:
+    accessModes:
+    - ReadWriteOnce
+    resources:
+      requests:
+        storage: 1Gi
+  backend:
+    externallyManaged: false
+  authSecret:
+    externallyManaged: false
+```
+
+#### supported versions
+`1.18.0`
+
+# RabbitMQ
+
+This release also brings support for provisioning **RabbitMQ**, a popular message queue tool that connects applications to exchange messages, facilitating scalability and stability for a competitive edge. Support for provisioning, Custom configuration via kubernetes secrets and preinstalled management UI has been added in this release.
+
+Here's a sample manifest to provision RabbitMQ.
+
+```yaml
+apiVersion: kubedb.com/v1alpha2
+kind: RabbitMQ
+metadata:
+  name: rabbit-dev
+  namespace: demo
+spec:
+  version: "3.12.12"
+  replicas: 3
+  storage:
+    accessModes:
+      - ReadWriteOnce
+    resources:
+      requests:
+        storage: 1Gi
+    storageClassName: standard
+  storageType: Durable
+  terminationPolicy: WipeOut
+```
+
+#### supported versions
+`3.12.12`
+
+# Elasticsearch
+
+ElasticsearchDashboard, a custom resource is used to provision Kibana for Elasticsearch cluster and Opensearch-Dashboards for Opensearch cluster has received a major change. The `apiVersion` for this CR has been updated to  `elasticsearch.kubedb.com/v1alpha1`. Here's a sample manifest to provision ElasticsearchDashboard.
+
+```yaml
+apiVersion: elasticsearch.kubedb.com/v1alpha1
+kind: ElasticsearchDashboard
+metadata:
+  name: es-cluster-dashboard
+  namespace: demo
+spec:
+  enableSSL: true
+  databaseRef:
+    name: es
+  terminationPolicy: WipeOut
+```
+
+# Postgres
+
+### Grafana Alerts Dashboard
+In this release, we bring support for Postgres grafana summary dashboard with alerts for a specific Postgres instance.
+
+In order to use the alert dashboards, You need to do the following:
```

Comment:
In order to use the alert dashboards, You need to do the following: 

To use the alert dashboards :

Reply:
replaced